### ISSUE TITLE
Update status message should not count disabled packages

### ISSUE THREAD

**Issue by KSXGitHub** [atom/settings-view#956]
Moved from https://github.com/atom/settings-view/issues/955

> Disabled packages should not be counted (`(N) Updates` message at the bottom-right corner)

**Comment by JonatanPaalsson** [comment]
Hey, I'm a new developer trying to resolve this issue and I have some questions. I've been looking at two different types of solutions and I wanted to ask here to see what you think.

1. If the disabled packages are not counted in the `package-updates-status-view` then it should resolve this issue with some work.
2. The other solution is by not adding disabled packages to `updatablePackages`inside `package-manager` this would result in that you only see your active packages in the Available Updates panel.

Do you want to see that disabled packages have updates? Do we want to update disabled packages when pressing update all?

**Comment by KSXGitHub** [comment]
@JonatanPaalsson 

> Do you want to see that disabled packages have updates?

Yes, but not as visible as enabled packages. Disabled packages should be hidden behind an expand button.

> Do we want to update disabled packages when pressing update all?

Normally, "Update All" will only update enabled packages, unless the user click the expand button, "Update All" will update all.

**Comment by JonatanPaalsson** [comment]
I just made a pull request to fix this. It does not have any specs yet, but other than that, I think it will solve this bug.

**Comment by JonatanPaalsson** [comment]
> Normally, "Update All" will only update enabled packages, unless user click the expand button, "Update All" will update all.

Maybe an "Update Active Packages"-button could be added as a separate feature?

**Comment by KSXGitHub** [comment]
@JonatanPaalsson makes sense

**Comment by JonatanPaalsson** [comment]
@KSXGitHub I'd be happy to give it a shot if new issue is created
